### ISSUE TITLE
 Error creating machine: Get https://api.github.com/repos/boot2docker/boot2docker/releases: dial tcp: lookup api.github.com: no such host

### ISSUE THREAD

**Issue by roldancer** [docker/machine#435]
Hi All,
    I'm testing the docker machine tool, it looks really good, yesterday I could create a machine using VirtualBox driver, the image boot2docker.iso is stored in my filesystem.
Now I trying to create a new virtualbox machine, but I don't have internet acces from the current network, so my question is, Is there any parameter to say "ehhh don't ask for the latest image available, use my current image that is stored in my defaul location" ?

This is the error that I get when i try to create the machine:

 ./docker-machine create -d virtualbox dockerDEV2
ERRO[0000] Error creating machine: Get https://api.github.com/repos/boot2docker/boot2docker/releases: dial tcp: lookup api.github.com: no such host 
WARN[0000] You will want to check the provider to make sure the machine and associated resources were properly removed. 
FATA[0000] Error creating machine

Basically the idea is to force docker machine to use my downloaded image, is it possible ?

Thanks

**Comment by sthulb** [comment]
#424 should fix this, after that is accepted, we should keep a local repo of b2d ISO files in ~/.docker/machine/.b2d or something :)

**Comment by ehazlett** [comment]
We already store the ISO locally. That error is checking to see if it's the
latest. We can change that to a warn and continue with the cached ISO if
it's available.

**Comment by roldancer** [comment]
That would be excelent, also the create command could have an aditional flag such us "--ignoreISOUpdate", if this flag is in the cmd then don't chech for updates ... what do you think ?

**Comment by nathanleclaire** [comment]
Oh interesting, I think we could modify #442 to also include a test for this case.
